rpg.py

```diff
@@ -153,8 +148,8 @@ def run(self):
                         print(player.name + ':', player.hp)
             if acting_player.is_alive and acting_player.get_all_enemies(self.players):     
                 self.add_into_queue(acting_player, current_game_time)
-            else:
-                print(f'{acting_player.name} is dead!')
+        else:
```

Comment:
This is not correct python. The indent is wrong.

You need to think about all the possible cases when you make an if statement.

In this case, `if acting_player.is_alive and acting_player.get_all_enemies(self.players): ` implies the follow up questions:
What should the program do if `not acting_player.is_alive and acting_player.get_all_enemies(self.players)`, what should it do if `acting_player.is_alive and not acting_player.get_all_enemies(self.players)`, and what should it do if `not acting_player.is_alive and not acting_player.get_all_enemies(self.players)`

Reply:
Well, If the player is alive, and has something to do, Great, add it back to the queue, if there's not in any of those statements, I want it to do nothing, the problem is it's adding back to the queue regardless of the if statement

Reply:
Then implement that.